internal/backends/winit/renderer/skia.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+use std::{
+    cell::RefCell,
+    rc::{Rc, Weak},
+};
+
+use i_slint_core::{
+    graphics::rendering_metrics_collector::RenderingMetricsCollector, item_rendering::ItemCache,
+};
+
+mod itemrenderer;
+mod textlayout;
+
+pub struct SkiaRenderer {
+    window_weak: Weak<i_slint_core::window::WindowInner>,
+}
+
+impl super::WinitCompatibleRenderer for SkiaRenderer {
+    type Canvas = SkiaCanvas;
+
+    fn new(window_weak: &std::rc::Weak<i_slint_core::window::WindowInner>) -> Self {
+        Self { window_weak: window_weak.clone() }
+    }
+
+    fn create_canvas_from_glutin_context(
+        &self,
+        gl_context: &glutin::WindowedContext<glutin::PossiblyCurrent>,
+        winsys_name: Option<&str>,
+    ) -> Self::Canvas {
+        let _platform_window = gl_context.window();
```

Comment:
Does this call have some side effects that are needed?

Reply:
That's indeed an unused variable now - thanks for spotting!